src/app/code/community/MageHackDay/TwoFactorAuth/Helper/Data.php

```diff
@@ -21,4 +21,9 @@ public function getRememberMeDuration()
     {
         return Mage::getStoreConfig('admin/security/remember_me_duration');
     }
+
+    public function getAllowIps()
+    {
+        return Mage::getStoreConfig('admin/security/allow_ips');
```

Comment:
These really should be CIDR notation otherwise a whole block would require 256 entries.  http://stackoverflow.com/a/16046469/582138 gives an overview of how to convert CIDR to range